lib/slack-ruby-bot/commands/base.rb

```diff
@@ -1,7 +1,69 @@
 module SlackRubyBot
   module Commands
+
+    # class methods to modify the Commands::Base class. All methods are related to
+    # providing before/after hooks to commands.
+    module CommandHooks
+      def remove_command_hooks(type = nil, name = nil)
```

Comment:
This seems to be only used in tests, so sounds like this should be some kind of a more straightforward `reset!` that resets `@hooks` or something like that.